data_structures/src/radon_report.rs

```diff
@@ -62,7 +84,7 @@ where
 
 /// This trait identifies a RADON-compatible type system, i.e. most likely an `enum` with different
 /// cases for different data types.
-pub trait TypeLike: std::marker::Sized {
+pub trait TypeLike: std::clone::Clone + std::marker::Sized {
```

Comment:
No need to be explicit:

```suggestion
pub trait TypeLike: Clone + Sized {
```